### ISSUE TITLE
Camera Feed Not Shown In Microsoft Edge

### ISSUE THREAD

**Issue by moriel5** [gokulkrishh/qrcodescan.in#12]
OS: Linux (Solus)
Browsers: Firefox (Nightly), Edge (Dev)
Webcam: Logitech B525

Hello, I usually use QR Code Scanner as a PWA on my desktop and laptop, however lately I have noticed that the camera Feed simply does not show up, with only a grey background appearing.

From my testing, QR Code Scanner simply does not work on Edge, however it does work on Firefox (which does not support PWAs), and other sites (such as Microsoft Teams or Jitsi, as well as an assortment of webcam testing sites, work just fine on Edge.

**Comment by gokulkrishh** [comment]
@moriel5 I haven't testing this out in Linux. If the edge is chromium based it should work as expected.

Can you let me know if the Egde browser you are using in chromium based or not?

**Comment by moriel5** [comment]
Yeah, Edge on Linux cannot be anything but Chromium-based, as that was what had allowed it to be cross-platform in the first place.

So yes, it is Chromium-based.

**Comment by gokulkrishh** [comment]
@moriel5 Not sure how to reproduce this as I don't have linux system. Can you try out now and tell me this issue still exists or not. Thanks

**Comment by moriel5** [comment]
Currently this needs to be put on hold, as I had reinstalled Solus, and still need to package Microsoft Edge.

**Comment by gokulkrishh** [comment]
@moriel5 Alright, keep us posted. Thanks.

**Comment by moriel5** [comment]
No problem, this will take some time, however, as I have other tasks that are more urgent, and I an going to need to prepare the package.yml (the "recipe" for packaging) for Edge from scratch (I accidentally wiped all of my package.ymls and was unable to restore them).

**Comment by hi-lay** [comment]
OS: Android
Browsers: Edge Canary

It can't be used on edge android, but I can see that the camera is already turned on through access dots. I tried other online scaners to use the camera. but qrcodesan.in not work.

test url : https://www.online-qr-scanner.com/

**Comment by gokulkrishh** [comment]
@hi-lay Can you try this in the stable browser instead of Canary and let me know?

**Comment by hi-lay** [comment]
> @hi-lay Can you try this in the stable browser instead of Canary and let me know?

I tried the edge standard version and had the same problem.
Edge Version: 93.0.961.53

**Comment by hi-lay** [comment]
> @hi-lay Can you try this in the stable browser instead of Canary and let me know?



> > @hi-lay Can you try this in the stable browser instead of Canary and let me know?
> 
> I tried the edge standard version and had the same problem.
> Edge Version: 93.0.961.53

The upper left corner of the picture represents that the camera is being used and is detected by Access Dots.

![image](https://user-images.githubusercontent.com/13347954/134794967-e21b7ad2-ddb8-42a0-915c-30be543a5c90.png)

**Comment by gokulkrishh** [comment]
@hi-lay I see got it. Not sure why its not working. I will check it out. Thanks for reporting.
